### PR TITLE
add ompi_autogen_arguments to OMPI Install module

### PR DESCRIPTION
**Pull request by ggouaillardet**

### CODE REVIEW

**Comment by ggouaillardet** [comment]
@jsquyres can you please review this ?
once this is merged and mtt on cisco clusters is updated, you will need to update your .ini
```patch
diff --git a/cisco/mtt/community/cisco-ompi-core-testing-master.ini b/cisco/mtt/community/cisco-ompi-core-testing-master.ini
index 5bb48cb..1172c83 100644
--- a/cisco/mtt/community/cisco-ompi-core-testing-master.ini
+++ b/cisco/mtt/community/cisco-ompi-core-testing-master.ini
@@ -143,6 +143,7 @@ include_section = Configure limited MPI install
 env_module = cisco/autotools/ac269-am1131-lt242
 
 ompi_autogen = 1
+ompi_autogen_arguments = --force
 ompi_configure_arguments = @common_config_args@
 
 #----------------------------------------------------------------------
```

File: lib/MTT/MPI/Install/OMPI.pm [hunk 1]
```diff
@@ -78,6 +80,11 @@ sub Install {
     $config->{autogen} = $tmp
         if (defined($tmp));
 
+    $tmp = Value($ini, $section, "ompi_autogen_arguments");
+    $tmp =~ s/\n|\r/ /g;
```
Comment (jsquyres) [inline lib/MTT/MPI/Install/OMPI.pm]: Should this be:

```perl
$tmp =~ s/[\n\t]/ /g;
```

Comment (ggouaillardet) [inline lib/MTT/MPI/Install/OMPI.pm]: honestly, i just copy/pasted the ```ompi_configure_arguments``` regex a few lines above.

```perl
     $tmp = Value($ini, $section, "ompi_configure_arguments");
     $tmp =~ s/\n|\r/ /g;
```

is there anything you want me to change ?

Comment (jsquyres) [inline lib/MTT/MPI/Install/OMPI.pm]: I guess we should disregard my comment; I was unfamiliar with the `|` perl regexp operator...

**Pull request merged**